shopify/customized-bundle-template/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "esModuleInterop": true,
+    "allowJs": true,
+    "noImplicitAny": true,
+    "sourceMap": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "forceConsistentCasingInFileNames": true,
+    "target": "es2020",
+    "lib": ["es2020", "DOM"],
+    "skipLibCheck": true,
+    "jsx": "react-jsx",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "baseUrl": "/",
```

This is invalid and the Gadget editor will throw a warning.
We're also missing the `path` definitions here
```suggestion
    "baseUrl": "/"
```